bitnami/redis/templates/health-configmap.yaml

```diff
@@ -9,15 +9,19 @@ metadata:
     release: {{ .Release.Name }}
 data:
   ping_readiness_local.sh: |-
+    #!/bin/bash
 {{- if .Values.usePasswordFile }}
     password_aux=`cat ${REDIS_PASSWORD_FILE}`
     export REDIS_PASSWORD=$password_aux
+{{- end }}
+{{- if .Values.usePassword }}
+    no_auth_warning=$([[ "$(redis-cli --version)" =~ (redis-cli 5.*) ]] && echo --no-auth-warning)
```

Comment:
Why don't you use `image.tag` parameter in cobination of `{{- semverCompare }}` function?

Reply:
IMO image.tag is an input from user and unpredictable. It's more reliable to check the `real` version of redis using cli.

Reply:
That's true, but using GoTemplates you don't need to execute `redis-cli` everytime you execute a readiness/liveness probe

Reply:
Finally I left the logic in the shell scripts.

Reply:
I think it is going to be easier to maintain when new versions of redis appear.